### PR TITLE
Revert of #120385

### PR DESCRIPTION
**Pull request by Jasguerrero**
Breaking google testing 
revert of: https://github.com/flutter/flutter/pull/120385
b/295065534

### CODE REVIEW

**Review by justinmc — APPROVED** [review]
LGTM :+1:  I will work on relanding this after this is merged.

File: packages/flutter/lib/src/widgets/navigator.dart [hunk 1]
```diff
@@ -3348,6 +3295,7 @@ class _NavigatorReplaceObservation extends _NavigatorObservation {
   }
 }
 
+<<<<<<< HEAD
```
Comment (justinmc) [inline packages/flutter/lib/src/widgets/navigator.dart:3298]: Ah nevermind, here's a merge conflict!  It looks like everything in the "HEAD" block here should be deleted.  It's all stuff from my PR.

**Review by justinmc — DISMISSED** [review]
Looks like there is one merge conflict, but it should be easy to fix.  I left a comment.

**Review by zanderso — APPROVED** [review]
lgtm on green

**Review by justinmc — APPROVED** [review]
LGTM :+1:

**Pull request merged**